boards/PDM/Inc/Io/Io_Efuse_22XS4200.h

```diff
@@ -0,0 +1,646 @@
+/**
```

Comment:
Did you have to fill in all the `MASK` and `ADDR` manually?  Or was it taken from somewhere?

Reply:
I manged to find a driver for a eval-board and I made modifications to that

Reply:
oh out of curiosity, can you share the link?

Reply:
Here's a direct download link:
[https://cache.nxp.com/secured/assets/downloads/en/embedded-components/36V-eXtremeSwitch-PEx-SW-EMC.zip?__gda__=1588974472_d7828465161180e0f29cae8414dc18bd&fileExt=.zip](https://cache.nxp.com/secured/assets/downloads/en/embedded-components/36V-eXtremeSwitch-PEx-SW-EMC.zip?__gda__=1588974472_d7828465161180e0f29cae8414dc18bd&fileExt=.zip)

It's for this eval-board:
[TWR-MC36XSDEVB](https://www.nxp.com/design/development-boards/analog-toolbox/embedded-component-36-v-tower-system-extreme-switch-module:TWR-MC36XSDEVB)